Gruntfile.js

```diff
@@ -126,8 +126,8 @@ module.exports = function(grunt) {
       },
       test: {
         options: {
-          user: couchConfig.username,
-          pass: couchConfig.password,
+          user: 'admin',
+          pass: 'pass',
```

Comment:
What happens if we remove this `options` property altogether? Shouldn't it just work because we pass authentication in the URL ?

Reply:
It does work. I have removed it altogether.